clients/js/DEVELOP.md

```diff
@@ -0,0 +1,21 @@
+# Develop
+
+This readme is helpful for local dev.
+
+### Prereqs:
+- Make sure you have Java installed (for the generator)
+- Make sure you are running the docker backend at localhost:8000 (*there is probably a way to stand up the fastapi server by itself and programatically in the loop of generating this, but not prioritizing it for now. It may be important for the release)
+
+### Generating
+1. `yarn` to install deps
+2. `yarn genapi-zsh` if you have zsh
+3. `yarn start` will run the parcel server for easier testing. That is in the `example` folder.
```

Comment:
```suggestion
3. `yarn start` will run the parcel server for easier testing. That is in the `examples` folder.
```

Reply:
I think we should be clear that you run the respective examples in /browser or /node